java/graalvm/Dockerfile

```diff
@@ -45,10 +45,20 @@ ENV PATH $PATH:/usr/local/gcloud/google-cloud-sdk/bin
 # Install docker
 RUN yum install -y docker-engine docker-cli
 
-# Install terraform
+# Install Terraform (https://developer.hashicorp.com/terraform/install#linux)
+RUN yum install -y yum-utils
+RUN yum-config-manager --add-repo https://rpm.releases.hashicorp.com/RHEL/hashicorp.repo
+
+# https://support.hashicorp.com/hc/en-us/articles/26015918890259-HashiCorp-RPM-releases-404-on-RHEL
+RUN sed -i -e "s/\$releasever/7/g" "/etc/yum.repos.d/hashicorp.repo"
```

Comment:
Why do we need these three lines?

Reply:
It's following hashicorp's recommended way to install terraform. RHEL has additional logic needed to resolve the 404 issue and I'm following the guide here: https://support.hashicorp.com/hc/en-us/articles/26015918890259-HashiCorp-RPM-releases-404-on-RHEL